File: PooCC01CNetz/Semana05/Python/Ejercicios01/ArregloEmpleado.py
from Empleado import *
from Date import *

class ArregloEmpleado:
    def main(self):

        emps = [None] * 3

        emp1 = Empleado("Kevin", float(430), Date(12,8,2003))

        emp2 = Empleado()
        emp2.setName("Uriel")
        emp2.setSalary(float(340))
        emp2.setFecNac(Date(8,12,2005))

        emps[0] = emp1

        emps[1] = emp2

        emp3 = Empleado("Javier", float(380), Date(29, 3, 1982))
        emps[2] = emp3

        for empleado in emps:
            print("**************************************************")
            print("Nombre              : " + empleado.getName())
            print("Salario             : " + str(empleado.getSalary()))
            print("Fecha de Nacimiento : " + str(empleado.getFecNac().getDay())+ "/" + str(empleado.getFecNac().getMounth()) + "/" + str(empleado.getFecNac().getYear()))
            print("Quincena            : " + str(empleado.calcularQuincena()))

clase = ArregloEmpleado()
clase.main()

File: PooCC01CNetz/Semana05/Java/Ejercicios01/ArregloEmpleado.java
public class ArregloEmpleado {
    public static void main(String[] args) {

        Empleado[] emps = new Empleado[3];

        Empleado emp1 = new Empleado("Kevin", 430, new Date(12,8,2003));

        Empleado emp2 = new Empleado();
        emp2.setName("Uriel");
        emp2.setSalary(340);
        emp2.setFecNac(new Date(8,12,2005));

        emps[0] = emp1;

        emps[1] = emp2;

        Empleado emp3 = new Empleado("Javier", 380, new Date(29,3,1982));
        emps[2] = emp3;

        for (Empleado empleado : emps) {
            System.out.println("**************************************************");
            System.out.println("Nombre              : " + empleado.getName());
            System.out.println("Salario             : " + empleado.getSalary());
            System.out.println("Fecha de Nacimiento : " + empleado.getFecNac().getDay() + "/" + empleado.getFecNac().getMounth() + "/" + empleado.getFecNac().getYear());
            System.out.println("Quincena            : " + empleado.calcQuincena());
        }

    }
}   